.github/workflows/tests_suite.yml

```diff
@@ -75,6 +75,7 @@ jobs:
           - simulation/human_model_generation
           - perception/facial_expression_recognition
           - control/single_demo_grasp
+          - planning/end_to_end_planning
```

Comment:
Will this tool only work if installed from github? The `test-docker` job at least should work as well, no? (in the same workflow file, towards the end of the file)

Reply:
Done